### PR TITLE
Add example

### PR DESCRIPTION
**Pull request by grzegorz8**

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- a Jupyter Notebook container.
+
+Flink UI can be accessed at `http://localhost:8081`.
+Minio UI can be accessed at `http://localhost:9001` (`user`/`password`).
+Jupyter UI can be accessed at `http://localhost:8888`.
+
+## Build
+
+1. Build java artifacts first.
+   ```bash
```
Comment (andrzejdackiewicz) [inline example/README.md:25]: For everything to go smoothly, we need to be in flink-elastic-catalog directory, not in example directory.
Add `cd ../` before running `mvn clean install` ...
Besides that LGTM.

**Review by andrzejdackiewicz — APPROVED** [review]
LGTM!

File: example/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# example
+
```
Comment (andrzejdackiewicz) [inline example/README.md:2]: I would also add a very short description about this instruction. Something like this:
This repository contains definition docker-compose environment which makes it possible to use Jupyter Notebook using the Flink JDBC Elastic Catalog. ...

File: example/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Flink UI can be accessed at `http://localhost:8081`.
+Minio UI can be accessed at `http://localhost:9001` (`user`/`password`).
+Jupyter UI can be accessed at `http://localhost:8888`.
+
+## Build
+
+1. Build java artifacts first.
+   ```bash
+   mvn clean install -DskipTests -T 1C -f flink-connector-jdbc/pom.xml
```
Comment (andrzejdackiewicz) [inline example/README.md:26]: We can add information in point 1 that the first command builds the **flink-connector-jdbc**.
Second instruction builds an additional dependency **flink-elastic-catalog**.

File: example/README.md (hunk 1 above)
Comment (grzegorz8) [reply example/README.md:25]: fixed

File: example/README.md (hunk 3 above)
Comment (grzegorz8) [reply example/README.md:26]: fixed

File: example/README.md (hunk 2 above)
Comment (grzegorz8) [reply example/README.md:2]: done

**Pull request merged**